tck/tck-dist/src/main/assembly/assembly.xml

```diff
@@ -19,68 +19,43 @@
 
 <assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
-   <baseDirectory>jsonp-tck</baseDirectory>
-   <formats>
-      <format>zip</format>
-   </formats>
-
-   <files>
-      <!-- license -->
-      <file>
-         <source>LICENSE_${license}.md</source>
-         <destName>LICENSE.md</destName>
-      </file>
-      <file>
-         <source>artifact-install.sh</source>
-         <outputDirectory>artifacts</outputDirectory>
-      </file>
-   </files>
-   <fileSets>
-         <fileSet>
-            <directory>../tck-docs/</directory>
-            <outputDirectory>docs/</outputDirectory>
-            <includes>
-                <include>*.*</include>
-            </includes>
-         </fileSet>
-         <fileSet>
-            <directory>../tck-docs/userguide/target/generated-docs</directory>
-            <outputDirectory>docs/pdf-usersguide</outputDirectory>
+    <id>distribution</id>
+    <baseDirectory>jsonp-tck</baseDirectory>
+    <formats>
+        <format>zip</format>
```

Comment:
Fyi: TCK in EE10 is acceptable in a form of jar, too, AFAIK.

Reply:
https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/ shows only zip files

Reply:
Right, I guess Jakarta REST would be the first one...

Reply:
my thinking is that a `.jar` is something I can put on the classpath and/or run directly which is not true with this particular artifact - at least not yet. We could certainly provide a `.jar` containing tests only (say current `jakarta.json:jakarta.json-tck-tests` artifact) and a `.zip` distribution for offline use (with the userguide etc)